src/citrine/resources/predictor.py

```diff
@@ -397,115 +367,3 @@ def create_default(self,
                 "prefer_valid": prefer_valid}
         data = self.session.post_resource(path, json=body, version=self._api_version)
         return self.build(Predictor.wrap_instance(data["instance"]))
-
-    def convert_to_graph(self,
```

Comment:
I'm not sure we want to remove access to this endpoint. We still have customers with unarchived simple ML predictors in unarchived design workflows. I count about 150 such predictors across 5 customers.

So while the SDK can drop support for simple ML predictors, we should still allow them to convert for the time being.